doc/userdoc/guides/nest2_to_nest3/nest3_features/nest3_node_handles.rst

```diff
@@ -0,0 +1,353 @@
+.. _nodeid:
+
+New functionality for node handles (neurons and devices)
+========================================================
+
+In NEST 3.0, ``nest.Create()`` returns a *NodeCollection* object instead of a list of global IDs.
+This provides a more compact and flexible way for handling nodes.
+
+In most use cases, you will not need to make any changes to your scripts in NEST 3.0, unless you have used **topology** or **subnets**.
```

Comment:
I think many changes is better now, as people need to change to `synapse_model` and models like `spike_detector` is renamed.
```suggestion
In most use cases, you will not need to make many changes to your scripts in NEST 3.0, unless you have used **topology** or **subnets**.
```